sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.props

```diff
@@ -14,11 +14,18 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <!--Our source generators rely on the FunctionsExecutionModel property.-->
     <FunctionsExecutionModel>isolated</FunctionsExecutionModel>
     <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <DefaultExcludesInProjectFolder>$(DefaultExcludesInProjectFolder);host.json;local.settings.json</DefaultExcludesInProjectFolder>
   </PropertyGroup>
 
   <!--Enable Azure Functions project capability to enable tools-->
   <ItemGroup>
-        <ProjectCapability Include="AzureFunctions"/>
+    <ProjectCapability Include="AzureFunctions"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="host.json" CopyToOutputDirectory="PreserveNewest" CopyToPublisDirectory="PreserveNewest" />
+    <None Include="local.settings.json" CopyToOutputDirectory="PreserveNewest" CopyToPublisDirectory="Never" />
```

Comment:
I didn't know we could do `CopyToPublisDirectory="Never"` as attributes instead of the child element. When I tested this approach, I see the `local.settings.json` copied to my out directory. Is that not what you see?

Reply:
I have a typo thats why 😆 

But yes, attributes and child elements for items are equivalent in MSBuild.

Reply:
Fixed the spelling mistake